ACMAS/app/acmas_tables.sql

```diff
@@ -0,0 +1,60 @@
+create table "ACMAS_Web_university"
+(
+    id   bigserial
+        primary key,
+    name varchar(50) not null
+);
+
+alter table "ACMAS_Web_university"
+    owner to hello_acmas;
+
+create table "ACMAS_Web_course"
+(
+    id         bigserial
+        primary key,
+    name       text        not null,
+    code       varchar(20) not null,
+    university text        not null,
+    semster    varchar(30) not null,
```

Comment:
typo: semester

Reply:
Yeah, unfortunately, this is a pre-existing typo in the previous db. I tried to fix it (see dead code) but fixing it caused a crazy number of issues that are best suited for their own issue/PR.